docs/cse/records-signals-entities-insights/create-an-entity-group.md

```diff
@@ -92,12 +92,15 @@ Follow these instructions to create an Entity Group that corresponds to a group
 1. **Inventory Type**. Select one of:
     * Computer
     * User
+1. **Inventory Key**. Select an attribute to use from the **Inventory Type** selected above. Select **groups** if you want to use an existing Entity Group attribute. 
 1. **Source**. Select an inventory source from the pull-down list.
-1. **Group**. Enter the name of the group in the inventory system that contains the entities you want to add to the Entity Group.
+1. **Value**. Enter a value for the attribute selected in the **Inventory Key** field above. If **groups** was selected in the **Inventory Key** field, enter the name of the group in the inventory system that contains the entities you want to add to the Entity Group. (If you select the **Dynamic Schema Tags** checkbox below, you must enter *****.) 
```

Comment:
Small nit-pick:
Reads: (If you select the **Dynamic Schema Tags** checkbox below, you must enter *****.)

When `Dynamic Schema Tags` is selected, `*` will automatically be applied as stated correctly below on line 99: `If you select this option, the **Value** field changes to *****`. 

Might want to update this line, as it implies the user might be able to change this value when they cannot, and `*` is only a valid value when `dynamic schema tags` is checked.

Reply:
@AngelATX, per our Slack conversation, I removed 
>(If you select the **Dynamic Schema Tags** checkbox below, you must enter *)